pystac/catalog.py

```diff
@@ -230,11 +230,11 @@ def add_child(
         child: Union["Catalog", Collection],
         title: Optional[str] = None,
         strategy: Optional[HrefLayoutStrategy] = None,
-        keep_parent: bool = False,
+        set_parent: bool = True,
     ) -> None:
         """Adds a link to a child :class:`~pystac.Catalog` or
         :class:`~pystac.Collection`. This method will set the child's parent to this
-        object (except if a parent is set and keep_parent is true).
+        object (unless ``set_parent`` is False).
```

Comment:
Should we mention here that the self link may also get updated if the parent it set?